exciting_environments/mass_spring_damper/mass_spring_damper_env.py

```diff
@@ -15,121 +18,179 @@ class MassSpringDamper(core_env.CoreEnvironment):
     Action Variable:
         ``['force']``
 
-    Observation Space (State Space):
-        Box(low=[-1, -1], high=[1, 1])    
-
-    Action Space:
-        Box(low=-1, high=1)
-
     Initial State:
         Unless chosen otherwise, deflection and velocity is set to zero.
 
     Example:
         >>> import jax
+        >>> import jax.numpy as jnp
         >>> import exciting_environments as excenvs
+        >>> from exciting_environments import GymWrapper
         >>> 
         >>> # Create the environment
-        >>> env= excenvs.make('MassSpringDamper-v0',batch_size=2,d=2,k=0.5,max_force=10)
+        >>> msd=excenv.MassSpringDamper(batch_size=4,action_constraints={"force":10})
         >>> 
-        >>> # Reset the environment with default initial values
-        >>> env.reset()
+        >>> # Use GymWrapper for Simulation (optional)
+        >>> gym_msd=GymWrapper(env=msd)
         >>> 
-        >>> # Sample a random action
-        >>> action = env.action_space.sample(jax.random.PRNGKey(6))
+        >>> # Reset the environment with default initial values
+        >>> gym_msd.reset()
         >>> 
         >>> # Perform step
-        >>> obs,reward,terminated,truncated,info= env.step(action)
+        >>> obs,reward,terminated,truncated,info= gym_msd.step(action=jnp.ones(4).reshape(-1,1))
         >>> 
 
     """
 
-    def __init__(self, batch_size: int = 8, d: float = 1, k: float = 100, m: float = 1,  max_force: float = 20, solver=diffrax.Euler(), reward_func=None, tau: float = 1e-4, constraints: list = [10, 10]):
+    def __init__(
+        self,
+        batch_size: int = 8,
+        physical_constraints: dict = {"deflection": 10, "velocity": 10},
+        action_constraints: dict = {"force": 20},
+        static_params: dict = {"k": 100, "d": 1, "m": 1},
+        solver=diffrax.Euler(),
+        reward_func=None,
+        tau: float = 1e-4,
+    ):
         """
         Args:
             batch_size(int): Number of training examples utilized in one iteration. Default: 8
-            d(float): Damping constant. Default: 1
-            k(float): Spring constant. Default: 100
-            m(float): Mass of the oscillating object. Default: 1
-            max_force(float): Maximum force that can be applied to the system as action. Default: 20
-            reward_func(function): Reward function for training. Needs Observation-Matrix and Action as Parameters. Default: None (default_reward_func from class) 
+            physical_constraints(jdc.pytree_dataclass): Constraints of physical states of the environment.
```

Comment:
same as for the cartpole. We are expecting a dictionary, but the documentation tells us that it expects a dataclass. I understand where this comes from, but this needs to be acknowledged IMO.